en/docs/reference/mediators/enrich-mediator.md

```diff
@@ -22,98 +19,132 @@ The main properties of the Enrich Mediator are as follows:
-        For the target type ' `envelope` ', the action type should be `'replace` '. Herein, action type ' `child` ' is not acceptable because it adds an envelope within an envelope, and action type ' `sibling` ' is also not acceptable because there will be two envelopes in a message if you use it.
-  
--   **Type** - The type that the mediator uses from the original message to enrich the modified message that passes through the mediator.
-    -   **Custom** - Custom XPath value.
-    -   **Envelope** - Envelope of the original message used for enriching.
-    -   **Body** - Body of the original message used for enriching.
-    -   **Property** - Specifies a property. For information on how you can use the property mediator to specify properties, see [Property Mediator]({{base_path}}/reference/mediators/property-mediator).
-    -   **Inline** - Specifies a literal value for enriching.
-    -   **Key** - Specifies that the target type is key. Specifically used to rename an existing key name in JSON payloads. *(Supported for JSON only)*.
--   **XPath Expression** - Refer the [Source configuration](#source-configuration) above.
-
-    !!! Info
-        The target type depends on the source type. For the valid and invalid combinations of source and target types, see the below table.
-
-        <table>
-            <tr>
-                <th></th>
-                <th colspan="5" style="horizontal-align: middle; text-align: center;">Target type</th>
-            </tr>
-            <tr>
-                <th rowspan="6" style="vertical-align: middle; text-align: center;">Source type</th>
-                <th></th>
-                <th>custom</th>
-                <th>envelope</th>
-                <th>body</th>
-                <th>property</th>
-            </tr>
-            <tr>
-                <th>custom</th>
-                <td>valid</td>
-                <td>invalid</td>
-                <td>valid</td>
-                <td>valid</td>
-            </tr>
-            <tr>
-                <th>envelope</th>
-                <td>invalid</td>
-                <td>invalid</td>
-                <td>invalid</td>
-                <td>valid</td>
-            </tr>
-            <tr>
-                <th>body</th>
-                <td>valid</td>
-                <td>invalid</td>
-                <td>invalid</td>
-                <td>valid</td>
-            </tr>
-            <tr>
-                <th>property</th>
-                <td>valid</td>
-                <td>valid</td>
-                <td>valid</td>
-                <td>valid</td>
-            </tr>
-            <tr>
-                <th>inline</th>
-                <td>valid</td>
-                <td>valid</td>
-                <td>valid</td>
-                <td>valid</td>
-            </tr>
-        </table>
+
+!!! Info
+    For the target type `envelope`, the action type should be `replace`. Both action type `child` and `sibling` are not acceptable because it adds an envelope within an envelope.
```

Comment:
```suggestion
    For the target type `envelope`, the action type should be `replace`. Both action types `child` and `sibling` are not acceptable because they add an envelope within an envelope.
```